apb.yml

```diff
@@ -15,7 +15,27 @@ metadata:
   bindingsLimit: 2
 plans:
   - name: default
-    description: Persistent deployment of mdc-server
+    description: This plan should be used in most use cases
+    default: true
```

Comment:
This "default" flag (which I found [here](https://github.com/ansibleplaybookbundle/mariadb-apb/blob/master/apb.yml#L25)), doesn't affect which plan is autoselected in the service catalog in OpenShift.

![Screenshot from 2019-03-11 17-25-15](https://user-images.githubusercontent.com/442386/54193895-ea764e00-44b2-11e9-9b3d-2014fe579449.png)

I have narrowed it down to alphabetical order of the `metadata.displayName` property. If it's important to us that the "Default" is selected first, then either we need to change the name of it to something that comes before "Custom" alphabetically, or rename "Custom" to something that comes alphabetically after "Default".